### PR TITLE
Show vpn state at brave://skus-internals

### PR DESCRIPTION
**Pull request by simonhong**
fix https://github.com/brave/brave-browser/issues/32030

Resolves 

## Submitter Checklist:

- [x] I confirm that no [security/privacy review is needed](https://github.com/brave/brave-browser/wiki/Security-reviews) and no other type of reviews are needed, or that I have [requested](https://github.com/brave/reviews/issues/new/choose) them
- [x] There is a [ticket](https://github.com/brave/brave-browser/issues) for my issue
- [x] Used Github [auto-closing keywords](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue) in the PR description above
- [x] Wrote a good [PR/commit description](https://google.github.io/eng-practices/review/developer/cl-descriptions.html)
- [x] Squashed any review feedback or "fixup" commits before merge, so that history is a record of what happened in the repo, not your PR
- [x] Added appropriate labels (`QA/Yes` or `QA/No`; `release-notes/include` or `release-notes/exclude`; `OS/...`) to the associated issue
- [x] Checked the PR locally:
  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run lint`, `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`
- [x] Ran `git rebase master` (if needed)

## Reviewer Checklist:

- [x] A security review [is not needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or a link to one is included in the PR description
- [x] New files have MPL-2.0 license header
- [x] Adequate test coverage exists to prevent regressions
- [x] Major classes, functions and non-trivial code blocks are well-commented
- [x] Changes in component dependencies are properly reflected in `gn`
- [x] Code follows the [style guide](https://chromium.googlesource.com/chromium/src/+/HEAD/styleguide/c++/c++.md)
- [x] Test plan is specified in PR before merging

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on
- [ ] All relevant documentation has been updated, for instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protections
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Custom-Headers
  - [ ] https://github.com/brave/brave-browser/wiki/Web-Compatibility-Exceptions-in-Brave
  - [ ] https://github.com/brave/brave-browser/wiki/QA-Guide
  - [ ] https://github.com/brave/brave-browser/wiki/P3A

## Test Plan:

1. Load brave://skus-internals/ and click `Fetch VPN State`
2. Check VPN specific order info is fetched.

### CODE REVIEW

**Comment by bsclifton** [comment]
😍 
image

File: browser/ui/webui/skus_internals_ui.cc [hunk 1]
```diff
@@ -80,6 +80,79 @@ void SkusInternalsUI::GetSkusState(GetSkusStateCallback callback) {
...
+      }
+
+      if (auto* id = order_dict->FindString("id")) {
+        order_dict_output.Set("id", *id);
+      }
+      if (auto* expires_at = order_dict->FindString("expires_at")) {
+        order_dict_output.Set("expires_at", *expires_at);
+      }
+    }
+    // Set output with env like {skus:prodiction: {...}}.
```
Comment (bsclifton) [inline browser/ui/webui/skus_internals_ui.cc:149]:
```suggestion
    // Set output with env like {skus:production: {...}}.
```

**Review by bsclifton — APPROVED** [review]
Changes look great! Also works perfectly 😄 
This will be extremely useful - thanks!

File: browser/ui/webui/skus_internals_ui.cc (hunk 1 above)
Comment (simonhong) [reply browser/ui/webui/skus_internals_ui.cc]: Done! 👍🏼

**Comment by stephendonner** [comment]
## Verified `PASSED` using 

```
Brave	1.59.48 Chromium: 116.0.5845.114 (Official Build) nightly (64-bit) 
Revision	ee007cd855091dd06de1fb87a21e31ffb040d88b
OS	Windows 10 Version 22H2 (Build 19045.3393)
```

### Steps:
1. installed `1.59.48`
2. launched Brave
3. purchased, configured, and connected to `Brave VPN`
4. confirmed connection via `WireGuard` which is now default on `nightly` 👍 
5. opened `brave://skus-internals`
6. confirmed new `VPN State:` field and `Fetch VPN state` button
7. clicked on `Fetch VPN state` button
8. confirmed `order_id` and other fields were present
9. compared the above with the `Local State` entries in my user profile directory
10. additionally confirmed toggle to collapse/expand further, as well as the existing `Fetch SKUs state` button

example | example | example | example | example
---------|----------|---------|----------|--------
image | image (1) | image (2) | image (4) | image (3)

**Pull request merged**